pkgs/build-support/writers/default.nix

```diff
@@ -178,6 +178,16 @@ rec {
   writeJSBin = name:
     writeJS "/bin/${name}";
 
+  writeNginxConfig = name: text: pkgs.runCommand name {
+    inherit text;
+    passAsFile = [ "text" ];
+  } /* sh */ ''
+    cp "$textPath" $out
+    ${pkgs.nginx-config-formatter}/bin/nginxfmt $out
+    ${pkgs.gnused}/bin/sed -i '/^$/d' $out
+    ${pkgs.gixy}/bin/gixy $out
```

Comment:
How likely is gixy to break valid configuration?

Reply:
Should there be a opt-out flag?

Reply:
I'm not sure how likely it is, but I'd argue that broken configurations should be fixed instead. Alternatively, `writeNginxConfig` can be overridden. This might not be the most convenient solution, but it's also not too painful either. Basically a `writeNginxConfig = self.writeText` would be sufficient. Anyway, if it's really desired, we can make it optional, or even pass arguments to `gixy` (but I haven't needed that myself before).

Reply:
I guess we could test on master and see how many people would complain about it.

Reply:
here to complain about it, breaks generic forward proxy configs